assessment.rb

```diff
@@ -2,11 +2,45 @@
 # Assessment answers to:
 
 # question 2:
+person = Licensee.new
+person.surname = 'Bishop'
+person.given_name = 'Caren'
+person.height = 62
+person.weight = 132
+person.dob = '1943-09-26'
+person.gender = 'm'
+person.eye_color = 'Brown'
+person.hair_color = 'Black'
+person.save ? puts('success') : puts('fail')
+
```

Comment:
This will work! There is, however, a shorter syntax for setting all of those properties at once...